AppCenter/AppCenter/Internals/Util/MSKeychainUtil.m

```diff
@@ -1,32 +1,34 @@
 #import <Foundation/Foundation.h>
 
-#import "MSKeychainUtil.h"
+#import "MSKeychainUtilPrivate.h"
 #import "MSUtility.h"
 
 @implementation MSKeychainUtil
 
-static NSString *AppCenterKeychainServiceName(void) {
+static NSString *AppCenterKeychainServiceName(NSString *suffix) {
   static NSString *serviceName = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
-    serviceName = [NSString stringWithFormat:@"%@.AppCenter", [MS_APP_MAIN_BUNDLE bundleIdentifier]];
+    serviceName = [NSString stringWithFormat:@"%@.%@", suffix, [MS_APP_MAIN_BUNDLE bundleIdentifier]];
```

Comment:
Oops my bad, it's the other way around, suffix should be the last. Changing it now.

Reply:
Done